sdk/cosmos/azure-cosmos/azure/cosmos/_routing/routing_range.py

```diff
@@ -226,3 +226,27 @@ def is_subset(self, parent_range: 'Range') -> bool:
         normalized_child_range = self.to_normalized_range()
         return (normalized_parent_range.min <= normalized_child_range.min and
                 normalized_parent_range.max >= normalized_child_range.max)
+
+class PartitionKeyRangeWrapper(object):
+    """Internal class for a representation of a unique partition for an account
+    """
+
+    def __init__(self, partition_key_range: Range, collection_rid: str) -> None:
+        self.partition_key_range = partition_key_range
+        self.collection_rid = collection_rid
+
+
+    def __str__(self) -> str:
+        return (
+            f"PartitionKeyRangeWrapper("
+            f"partition_key_range={self.partition_key_range}, "
+            f"collection_rid={self.collection_rid}, "
+        )
+
+    def __eq__(self, other):
```

Comment:
What are the `_eq` semantics when `partition_key_range` or `collection_rid` are null?